litmus-portal/frontend/src/pages/MonitoringDashboardPage/styles.ts

```diff
@@ -66,50 +58,158 @@ const useStyles = makeStyles((theme) => ({
 
   // Menu option
   menuItem: {
+    minWidth: '10rem',
+    height: '2.5rem',
+  },
+
+  menuItemSelected: {
+    background: theme.palette.primary.light,
+    color: theme.palette.secondary.contrastText,
     '&:hover': {
       background: theme.palette.primary.light,
-      color: theme.palette.secondary.contrastText,
     },
     minWidth: '10rem',
     height: '2.5rem',
   },
 
   expDiv: {
     display: 'flex',
-    flexDirection: 'row',
   },
 
   btnText: {
     fontWeight: 500,
   },
 
-  icon: {
-    width: '6rem',
-    height: '6rem',
+  selectDate: {
+    display: 'flex',
+    height: '2.9rem',
+    minWidth: '9rem',
+    marginLeft: theme.spacing(3.75),
+    border: `0.1px solid ${theme.palette.border.main}`,
+    background: theme.palette.background.paper,
+    borderRadius: 4,
+    textTransform: 'none',
+  },
+
+  displayDate: {
+    width: '100%',
+    color: theme.palette.text.primary,
+  },
+
+  rangeSelectorIcon: {
+    width: '0.625rem',
+    height: '0.625rem',
+    marginLeft: theme.spacing(1),
+    marginRight: theme.spacing(-1),
   },
 
-  modalHeading: {
-    marginTop: theme.spacing(3.5),
-    fontSize: '2.25rem',
-    marginBottom: theme.spacing(4.5),
+  rangeSelectorClockIcon: {
+    width: '0.825rem',
+    height: '0.825rem',
+    marginLeft: theme.spacing(-1),
+    marginRight: theme.spacing(1),
   },
 
-  modalBody: {
-    marginBottom: theme.spacing(4.5),
+  rangeSelectorPopover: {
+    marginTop: theme.spacing(37.5),
   },
 
-  flexButtons: {
+  formControl: {
+    width: '9rem',
+    marginLeft: theme.spacing(1.5),
+    [theme.breakpoints.down('sm')]: {
+      marginLeft: theme.spacing(3.75),
+    },
+    '& .MuiSelect-outlined': {
+      padding: '0.925rem',
+      '&:focus': {
+        borderColor: theme.palette.primary.main,
+      },
+      '& .MuiInputLabel-root': {
+        color: `${theme.palette.text.hint} !important`,
+        marginTop: `${theme.spacing(2)} !important`,
+      },
+    },
+  },
+
+  inputLabel: {
+    color: theme.palette.text.hint,
+    marginTop: theme.spacing(-1),
```

Comment:
These input things here...does the input field in litmus-ui not suffice the requirement?

Reply:
For the range slider its a external library, we will be build our own @arkajyotiMukherjee

Reply:
we do not yet have a select field in litmus-ui